engineering_tools/carma

```diff
@@ -339,6 +339,174 @@ carma__stop() {
+        docker run --rm -v carma-config-data:/opt/carma/vehicle/config --entrypoint sh busybox:latest -c \
+        'cat /opt/carma/vehicle/config/docker-compose-background-pc1.yml' | \
+        docker stack deploy --compose-file - carma-background
+
+
+        # Run part of the stack on a single host. Mostly for debugging purposes off of the dual PC setup
+        if [ "$2" = "single" ]; then 
+            echo "Deploying CARMA to single host Swarm stack..."
+
+            docker run --rm -v carma-config-data:/opt/carma/vehicle/config --entrypoint sh busybox:latest -c \
+            'cat /opt/carma/vehicle/config/docker-compose-pc1.yml' | \
+            docker stack deploy --compose-file - carma-pc1
+
+
+        # For use with two connected PCs with known IPs
+        # Saves the generated swarm join token and ssh's into the worker to execute and deploy
+        elif [ "$2" = "dual" ]; then
+            echo "Deploying CARMA to dual compute Swarm stack..."
+
+            local TKN=$(docker swarm join-token -q worker)
+            ssh dev@192.168.88.101 "docker swarm join --token $TKN 192.168.88.100:2377"
+
+            docker run --rm -v carma-config-data:/opt/carma/vehicle/config --entrypoint sh busybox:latest -c \
+            'cat /opt/carma/vehicle/config/docker-compose-pc1.yml' | \
+            docker stack deploy --compose-file - carma-pc1
+
+            docker run --rm -v carma-config-data:/opt/carma/vehicle/config --entrypoint sh busybox:latest -c \
+            'cat /opt/carma/vehicle/config/docker-compose-pc2.yml' | \
+            docker stack deploy --compose-file - carma-pc2
+            
+        
+        elif [ ! -z "$2" ]; then
+            echo "Unrecognized argument \"swarm deploy $2\""
+        fi
+
+
+    # Updates the carma-config-data volume based on a input tag string given by the user
+    elif [ "$1" = "update-config" ]; then
+        echo "Updating carma-config-data volume..."
+
+        if [ "$2" = "-d" ]; then
+            local USERNAME=usdotfhwastoldev
+            local TAG=$3
+        elif [ "$2" = "-c" ]; then
+            local USERNAME=usdotfhwastolcandidate
+            local TAG=$3
+        else
+            local USERNAME=usdotfhwastol
+            local TAG=$2
+        fi
+
+        if [[ -z $2 ]]; then
+            echo "Please specify a tag string to update carma-config-data volume."
+            echo "Done."
+            exit 1
+        fi
+
+        local IMAGE_NAME="$USERNAME/carma-config:$TAG"
+
+        if [ "$(docker images | grep $USERNAME/carma-config | grep $TAG )" ]; then
+            echo "Found image, applying to carma-config-volume"
+            docker run --rm --name carma-config-data -v carma-config-data:/opt/carma/vehicle/config -d $USERNAME/carma-config:$TAG
+        else
+            echo "Could not find $USERNAME/carma-config:$TAG in local images"
+        fi
+
+
+    elif [ "$1" = "edit-config" ]; then
```

Comment:
Similar to my previous comment on `carma swarm update-config`, could we change this to `config-edit` to remain consistent with the single PC non-swarm command (`carma config edit`)?